README.md

```diff
@@ -1,24 +1,45 @@
-# IDOM &middot; [![Tests](https://github.com/idom-team/idom/workflows/test/badge.svg)](https://github.com/idom-team/idom/actions?query=workflow%3ATest) [![PyPI Version](https://img.shields.io/pypi/v/idom.svg)](https://pypi.python.org/pypi/idom) [![License](https://img.shields.io/badge/License-MIT-purple.svg)](https://github.com/idom-team/idom/blob/main/LICENSE)
-
-IDOM connects your Python web framework of choice to a ReactJS frontend, allowing you to create **interactive websites without needing JavaScript!**
-
-Following ReactJS styling, web elements are combined into [reusable "components"](https://idom-docs.herokuapp.com/docs/guides/creating-interfaces/your-first-components/index.html#parametrizing-components). These components can utilize [hooks](https://idom-docs.herokuapp.com/docs/reference/hooks-api.html) and [events](https://idom-docs.herokuapp.com/docs/guides/adding-interactivity/responding-to-events/index.html#async-event-handlers) to create infinitely complex web pages.
-
-When needed, IDOM can [use components directly from NPM](https://idom-docs.herokuapp.com/docs/guides/escape-hatches/javascript-components.html#dynamically-loaded-components). For additional flexibility, components can also be [fully developed in JavaScript](https://idom-docs.herokuapp.com/docs/guides/escape-hatches/javascript-components.html#custom-javascript-components).
-
-Any Python web framework with Websockets can support IDOM. See below for what frameworks are supported out of the box.
-
-| Supported Frameworks | Supported Frameworks (External) |
-| --- | --- |
-|  [`Flask`, `FastAPI`, `Sanic`, `Tornado`](https://idom-docs.herokuapp.com/docs/guides/getting-started/installing-idom.html#officially-supported-servers) | [`Django`](https://github.com/idom-team/django-idom), [`Plotly-Dash`](https://github.com/idom-team/idom-dash), [`Jupyter`](https://github.com/idom-team/idom-jupyter) |
-
+# ReactPy &middot; [![Tests](https://github.com/reactive-python/reactpy/workflows/test/badge.svg)](https://github.com/reactive-python/reactpy/actions?query=workflow%3ATest) [![PyPI Version](https://img.shields.io/pypi/v/reactpy.svg)](https://pypi.python.org/pypi/reactpy) [![License](https://img.shields.io/badge/License-MIT-purple.svg)](https://github.com/reactive-python/reactpy/blob/main/LICENSE)
+
+ReactPy is a library for building user interfaces in Python without any Javascript.
+Interfaces made using ReactPy are composed of encapsulated components which look and
+behave similarly to those found in [ReactJS](https://reactjs.org/). These components are
+easy to create and reason about when you're getting started, as well as simple to extend
+and maintain when you need to make changes later.
+
+Whether you need to run a full blown application with a standalone server, or just want
+try things out in a Jupyter Notebook. ReactPy has you covered - it's both approachable
+for those without web development experience while also being powerful enough to grow
+with your ambitions.
```

Comment:
`Whether you need to run a full blown application with a standalone server, or just want
try things out in a Jupyter Notebook. ReactPy has you covered - it's both approachable
for those without web development experience while also being powerful enough to grow
with your ambitions.`

->

`ReactPy has you covered whether you need to run a full blown application with a standalone server, or simply want
try things out in [Jupyter](https://jupyter.org/). Designed with simplicity in mind, it can be used for those without web development experience while also being powerful enough to grow with your ambitions.`